lib/libshout/CMakeLists.txt

```diff
@@ -37,6 +35,9 @@ add_library(shout_mixxx STATIC
   src/tls.c
 )
 
+target_compile_definitions(shout_mixxx PRIVATE HAVE_CONFIG_H)
```

Comment:
This doesn't look like optimization flags. What about these changes?

Reply:
This replaces a line from above with individual, now duplicated flags:

 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DHAVE_CONFIG_H -Wall -ffast-math -pthread -g -O2")

I am not sure about the -pthread flag.
I have adopted this from the original build script. Is it required in the Mixxx context?